app/views/partials/trials-list.html

```diff
@@ -93,7 +95,7 @@
             {% endif %}
           {% endif %}
           {% if trial.isInProgress %}
-          <p>{{ trial.daysAfterStart }} days <span>until completed</span></p>
+          <p>{{ trial.daysBeforeCompletion }} days <span>until completed</span></p>
```

Comment:
I'm reviewing this only via the variable names. I imagine `daysAfterStart` was used here because when the trial is completed, `trial.isInProgress` would be false and it wouldn't fall in this clause anymore?